.contributing/the-process.md

```diff
@@ -75,16 +75,49 @@ Create: 
 
     -   A User Flow 
 
-        -   Step 1: Make a doc named: "x. *Use Case* < User Flow < *Section* (CL)"
+        1. Create a folder to hold all files needed to create a user flow
+            - Add title: "x. *Use Case* - User Flow files - *Section* (CL)"
+
+        2. Inside folder created in step 1, create a doc to outline action steps for the use case. 
+            - Add title: "x. *Use Case* - User Flow - *Section* (CL)"
+            - Add template to doc: 
+                -   A leaguer who...
+                    -   (context) 
+                -   wants to...
+                    -   (goal) 
+                -   so they...
+                    -   (action steps)
+            - After adding action steps, create "input task" in asana
+
+
+        3. Create lofi wireframe
+            - Sketch lofi wireframe on paper
+            - Create a google drawing to get input on sketch
+                - Add title: "x. *Use Case* - Lofi Sketch - *Section* (CL)"
+            - Take a photo and upload it to google drawing doc
+            - Add any helpful comments, e.g reasoning for design decisions, specific questions about design
+            - Create an "input task" in asana
```

Comment:
This is great - it's a chance to get explicit about these steps too I reckon (ie. the Asana part).  What if we said something like: 
- create a task in Asana with title "IS: Lofi Sketch < x. Use Case" and 
  - add a link to the google drawing in the task description 
  - move the task to the "Blockers & Reviews" column 
  - add a due date of tomorrow

Reply:
IS is Input Session, what do you think?

Reply:
@nickmeinhold love it, would it be good to have a key in the doc?
- like "IS = Input session" at the bottom of the doc

Reply:
and implemented your suggestion, thanks!